src/components/radios/index.md.njk

```diff
@@ -84,19 +84,23 @@ If one or more of your radio options is different from the others, it can help u
 
 ### Conditionally revealing content
 
-You can conditionally reveal content when the user selects a particular radio, so they only see content when it’s relevant to them.
+You can conditionally reveal a related question when the user selects a particular radio option, so they only see content when it’s relevant to them.
```

Comment:
do you need the word 'conditionally' here or is it implied by the rest of the sentence?

Reply:
Good point! I'd rather explain 'conditionally reveal' without resorting to saying it myself - I'd left this sentence alone but here's how I'd propose to edit it:
```suggestion
You can ask the user a related question when they select a particular radio option, so they only see the question when it's relevant to them.
```

Reply:
Looks good 👍